gpt_index/embeddings/google_use.py

```diff
@@ -0,0 +1,28 @@
+"""Google Universal Sentence Encoder Embedding Wrapper Module."""
+
+from typing import List
+
+import tensorflow_hub as hub
+
+from gpt_index.embeddings.base import BaseEmbedding
+
+# Google Universal Sentence Encode v5
+google_use = hub.load("https://tfhub.dev/google/universal-sentence-encoder-large/5")
```

Comment:
could this be a part of the `__init__` file? so it's lazily downloaded. Since I'd love to expose this module in `gpt_index/embeddings/__init__.py` (so users can do `from gpt_index.embeddings import GoogleUnivSentEncoderEmbeddings`)

Reply:
Sure, can you help me with that? Feel free to take the code and use it as you will.